src/ServiceControl.UnitTests/ApprovalFiles/APIApprovals.PublicClr.approved.txt

```diff
@@ -1174,6 +1175,7 @@ namespace ServiceControl.Recoverability
     public class FailedMessageGroupArchived : ServiceControl.Infrastructure.SignalR.IUserInterfaceEvent
     {
         public FailedMessageGroupArchived() { }
+        public string[] FailedMessagesIds { get; set; }
```

Comment:
How many IDs can be in this array? Any issues with transport message sizes? I don't see any bound checks so wondering what the max size is.

As this is text and often van contain GUID that is 36 characters, with quotes and comma its 39. Meaning, 128kb can contain roughly ~3.000 ID's assuming these are GUID's.

Reply:
Good catch @ramonsmits . We have added it to a number of contracts. If the number of items was bigger than the max allowed message size the publish would fail and the event would not be delivered. Given that we don't have a good way of checking it I'd say we ignore it for now and document architecture cruft here. 

Alternatively we could split one domain event into multiple on-the-wire events with portions of the data.

Reply:
Good point, Do we any pattern/guidance about that? Does that depend on the transport we use?

Reply:
Unfortunately depends on the transport :\

Reply:
We could add a new domain event when [creating retry batches](https://github.com/Particular/ServiceControl/blob/master/src/ServiceControl/Recoverability/Archiving/ArchiveAllInGroupHandler.cs#L66) and publish an integration event with message ids in that batch.

@seanfarmar do we need `GroupId` and `GroupName` in some of the integration events?

Reply:
@tmasternak For the scenario I'm thinking of, we don't because the customer is only tacking single messages...